k8s/namespaces/ccd/ccd-logstash/ccd-logstash.yaml

```diff
@@ -3,9 +3,8 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: ccd-logstash
-  namespace: ccd
   annotations:
-    flux.weave.works/automated: "true"
+    hmcts.github.com/prod-automated: disabled
```

Comment:
why's that?

Reply:
Do we want to roll it out to all environments when we import images ? I am guessing this will be a controlled change going through each env at a time.

Reply:
I guess yeah

Reply:
yeah needs to be controlled